From f77a5fd494e99227d8f2481fc52bd6039f093c17 Mon Sep 17 00:00:00 2001
From: Yuwei Chen <yuwei.chen@intel.com>
Date: Fri, 24 Jul 2020 10:34:44 +0800
Subject: [PATCH] Test case for Driver - FrontPageVfr.Vfr file

Description: Change the defined value of 'LABEL_FRANTPAGE_INFORMATION' in the MdeModulePkg/Application/UiApp/FrontPageVfr.Vfr
Repo: edk2
Signed-off-by: Yuwei Chen <yuwei.chen@intel.com>
---
 MdeModulePkg/Application/UiApp/FrontPageVfr.Vfr | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/MdeModulePkg/Application/UiApp/FrontPageVfr.Vfr b/MdeModulePkg/Application/UiApp/FrontPageVfr.Vfr
index aabf47fdf6..7798c40fa0 100644
--- a/MdeModulePkg/Application/UiApp/FrontPageVfr.Vfr
+++ b/MdeModulePkg/Application/UiApp/FrontPageVfr.Vfr
@@ -11,7 +11,7 @@
 
 #define FRONT_PAGE_FORM_ID             0x1000
 
-#define LABEL_FRANTPAGE_INFORMATION    0x1000
+#define LABEL_FRANTPAGE_INFORMATION    0x1013
 #define LABEL_END                      0xffff
 
 formset
-- 
2.27.0.windows.1

